src/agent/debuglet.ts

```diff
@@ -108,22 +108,19 @@ const formatBreakpoints = function(
 };
 
 export class Debuglet extends EventEmitter {
-  // TODO: Determine how to update the tests so that this can be private.
-  config_: DebugAgentConfig;
   private debug_: Debug;
   private v8debug_: V8DebugApi|null;
   private running_: boolean;
   private project_: string|null;
-  // TODO: Determine how to update the tests so that this can be private.
+  private debugletApi_: Controller;
+  private completedBreakpointMap_: {[key: string]: boolean};
+
+  // Exposed for testing
```

Comment:
I don't know if we necessarily need them to be private at this point as long as we have it documented in the code that they are public for testing purposes.